test/test_stateless.py

```diff
@@ -73,6 +73,7 @@ def test_functional_call_with_jit(self):
             self._run_call_with_mock_module(traced_module)
 
     @unittest.skipIf(not TEST_MULTIGPU, 'multi-GPU not supported')
+    @unittest.skip("This doesn't work right now")
```

Comment:
Can you open an issue to track this please?

Reply:
Done: https://github.com/pytorch/pytorch/issues/77576